cmake/modules/SwiftComponents.cmake

```diff
@@ -127,42 +127,30 @@ function(swift_is_installing_component component result_var_name)
   endif()
 endfunction()
 
-# swift_install_in_component(<COMPONENT NAME>
-#   <same parameters as install()>)
+# swift_install_in_component(<same parameters as install()>)
 #
 # Executes the specified installation actions if the named component is
 # requested to be installed.
 #
 # This function accepts the same parameters as install().
-function(swift_install_in_component component)
-  precondition(component MESSAGE "Component name is required")
+function(swift_install_in_component)
```

Comment:
Is the only big change here passing in COMPONENT?

Reply:
Yes, that's the only big change in this PR.